modules/apps/portal-security/portal-security-antisamy/build.gradle

```diff
@@ -145,8 +145,8 @@ private String _filterBlogsSanitizer(String line) {
 
 	if (line.contains('<literal value="hr"/>')) {
 		line = """\
-			|	${line}
-			|	<literal value="iframe"/>""".stripMargin()
+			|${line}
+			|				<literal value="iframe"/>""".stripMargin()
```

Comment:
Hey @thienquach87 
I just read Comments you shared to Brian.

So generally if there is some problems existing but not affecting your PR,
1. you can leave them as they were.
2. create a saperate ticket to report the problems and fix them if you want, under the new ticket number.
3. if the problem is really small, like a SF isuser in this case, you could fix them in your current PR as well, a saperate commit is prefered, so that reviewer(especially Brian who cares a lot about coding standards) can understand why you do it. If you have the non-related SF fix in the same PR, Brian would assume it's necessary for your current commit.

Reply:
Thank @arthurchan35 ,

 I updated source as your guide :)